src/current/_includes/releases/v23.2/v23.2.12.md

```diff
@@ -0,0 +1,77 @@
+- Show an alert message in the [DB Console]({% link v23.2/ui-overview.md %}) when the [license is expired]({% link v23.2/licensing-faqs.md %}#monitor-for-license-expiry) or if there is less than 15 days left before it expires. [#130509][#130509]
+- [DB Console]({% link v23.2/ui-overview.md %}) will show a notification alerting customers without an [Enterprise license]({% link v23.2/enterprise-licensing.md %}) to [upcoming license changes](https://www.cockroachlabs.com/enterprise-license-update/) with a link to more information. [#130509][#130509]
+
+<h3 id="v23-2-12-bug-fixes">Bug fixes</h3>
+
+- Declarative and legacy [schema changes]({% link v23.2/online-schema-changes.md %}) were incorrectly allowed to be executed concurrently before, which could lead to failing or hung schema change jobs. [#128838][#128838]
+- Fixed a bug that caused errors like `ERROR: column 'crdb_internal_idx_expr' does not exist` when accessing a table with an [expression index]({% link v23.2/expression-indexes.md %}) where the expression evaluates to an [`ENUM`]({% link v23.2/enum.md %}) type, e.g., `CREATE INDEX ON t ((col::an_enum))`. [#129092][#129092]
+- [Function]({% link v23.2/user-defined-functions.md %}) input parameters can no longer be of the `VOID` type. [#129281][#129281]
+- Internally issued queries that are not initiated within a [SQL session]({% link v23.2/show-sessions.md %}) no longer respect a statement timeout. This includes: [background jobs]({% link v23.2/show-jobs.md %}), queries issued by the [DB Console]({% link v23.2/ui-overview.md %}) that perform introspection, and the Cloud SQL shell. [#129517][#129517]
+- Fixed a bug where the `schema_locked` [storage parameter]({% link v23.2/with-storage-parameter.md %}) did not prevent a table from being referenced by a [foreign key]({% link v23.2/foreign-key.md %}). [#129753][#129753]
+- Users with the [`VIEWACTIVITY` SQL privilege]({% link v23.2/security-reference/authorization.md %}#viewactivity) can now request, view and cancel [statements bundles]({% link v23.2/explain-analyze.md %}) in the [DB Console]({% link v23.2/ui-overview.md %}). [#129803][#129803]
+- Fixed a bug where a [lease transfer]({% link v23.2/architecture/replication-layer.md %}#epoch-based-leases-table-data) could lead to a panic with the message `side-transport update saw closed timestamp regression`. The bug could occur when a node was overloaded and failing to [heartbeat its node liveness record]({% link v23.2/cluster-setup-troubleshooting.md %}#node-liveness-issues). [#129808][#129808]
+- Resolved a concerning [log message]({% link v23.2/logging-overview.md %}) that said `expiration of liveness record ... is not greater than expiration of the previous lease ... after liveness heartbeat`. This message is no longer possible. [#129808][#129808]
```

Comment:
We've published this note with this wording -- it's not clear to me what "resolved" means

```suggestion
- The [log message]({% link v23.2/logging-overview.md %}) `expiration of liveness record ... is not greater than expiration of the previous lease ... after liveness heartbeat` is no longer generated. [#129808][#129808]
```

Reply:
✅